awot/io/flight.py

```diff
@@ -274,7 +277,7 @@ def read_nasa_ames(filename, mapping_dict=None, platform=None):
             name_map = _get_name_map(platform)
         else:
             name_map = _make_name_map_from_varlist(hdr['VNAME'])
-
+    print(name_map)
```

Comment:
yes, just noticed that - will work it after lunch

On Wed, Jul 6, 2016 at 12:51 PM, Nick Guy <notifications@github.com> wrote:

> In awot/io/flight.py
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_nguy_AWOT_pull_39-23discussion-5Fr69778440&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=oh_IyBh3p6AhkrNBZyzmmDfeXg2WKrzfthPLOoHreWM&m=kLDgLpDcwDLlqvrfCyIjgANlBgnY8xUTD6TXI9wpi-g&s=MnhiEeU_Mcs7Kcpb4Xjer8YCB-rnZ9EyhZLENteJXeI&e=>
> :
>
> > @@ -274,7 +277,7 @@ def read_nasa_ames(filename, mapping_dict=None, platform=None):
> >              name_map = _get_name_map(platform)
> >          else:
> >              name_map = _make_name_map_from_varlist(hdr['VNAME'])
> > -
> > +    print(name_map)
>
> Any chance you would put an optional variable verbose in the
> read_nasa_ames function and have an optional printout of variable names?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_nguy_AWOT_pull_39_files_cb060f7bcfe4add71ae600444f603b2a390fa96b-23r69778440&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=oh_IyBh3p6AhkrNBZyzmmDfeXg2WKrzfthPLOoHreWM&m=kLDgLpDcwDLlqvrfCyIjgANlBgnY8xUTD6TXI9wpi-g&s=11ei73xSH1FqstnpkbYa402BuPAG7pqmNDkN4fltmPA&e=>,
> or mute the thread
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe_ADIuEOmG7amb1f7dJGCpeJcN9bL8Ywohks5qS-2DsQgaJpZM4JDHH1&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=oh_IyBh3p6AhkrNBZyzmmDfeXg2WKrzfthPLOoHreWM&m=kLDgLpDcwDLlqvrfCyIjgANlBgnY8xUTD6TXI9wpi-g&s=nBLjvmkD9ju5RWm7o3kxmzn7na9q4GLS7NHaOqX-K64&e=>
> .
>



-- 
Steve Nesbitt, Associate Professor
Department of Atmospheric Sciences, University of Illinois at
Urbana-Champaign
105 S. Gregory St., MC223, Urbana, IL 61801-3070  USA
voice : +1.217.244.3740  fax : +1.217.244.1752
internet : snesbitt@illinois.edu ;
https://www.atmos.illinois.edu/people/snesbitt ;
http://publish.illinois.edu/snesbitt